ci/docker/base.Dockerfile

```diff
@@ -76,4 +80,5 @@ RUN pyenv update && \
 
 ENV PATH="/root/.pyenv/shims:${PATH}"
 
-RUN pip install --upgrade pip setuptools wheel tox clang-format cupy-cuda11x
+ARG CUPY_PACKAGE=cupy-cuda11x
```

Comment:
Is cupy-cuda11x the default, and then overriden by the CUPY_PACKAGE env var in the case in which it runs on gh, making it install cupy-cuda12x?

Reply:
Correct. I thought I could leave some default value for all arguments.

Reply:
To be more precise, `CUPY_PACKAGE` is a docker build argument with default value `cupy-cuda11x`. When we build the base image for gh200, we set the build argument to `cupy-cuda12x`.